examples/ProviderUnlock/checkSignature.class.php

```diff
@@ -6,7 +6,7 @@
  *
  */
 
-class checkUrlSignature
+class CheckUrlSignature
```

Comment:
You should rename the file as well.

Reply:
You are right ;) Done, check the latest commit.